src/scss/components/_picture.scss

```diff
@@ -7,10 +7,68 @@
   }
 
   &--cover {
+    @include media-gt(small) {
+      -webkit-clip-path: polygon(0 0, 100% 0, 100% 85%, 50% 94%, 0 85%);
+      clip-path: polygon(0 0, 100% 0, 100% 85%, 50% 94%, 0 85%);
+    }
+    @include media-gt(medium) {
+      -webkit-clip-path: polygon(0 0, 100% 0, 100% 78%, 50% 97%, 0 78%);
+      clip-path: polygon(0 0, 100% 0, 100% 78%, 50% 97%, 0 78%);
+    }
+    @include media-gt(large) {
+      -webkit-clip-path: polygon(0 0, 100% 0, 100% 77%, 50% 94%, 0 77%);
+      clip-path: polygon(0 0, 100% 0, 100% 77%, 50% 94%, 0 77%);
+    }
+
     width: 100%;
     height: 100%;
     position: relative;
     overflow: hidden;
+    -webkit-clip-path: polygon(0 0, 100% 0, 100% 80%, 50% 94%, 0 80%);
+    clip-path: polygon(0 0, 100% 0, 100% 84%, 50% 100%, 0 84%);
+
+    &::before,
+    &::after {
+      width: 70%;
+      height: 220px;
+      position: absolute;
+      bottom: 0;
+      content: '';
+      background-color: get-color(air);
+      clip: rect(0, 6000px, 220px, 0);
+      transform-origin: center;
+      z-index: 1;
+    }
+
+    &::before {
```

Comment:
I guess that we have to do more work here in the arrow fallback, because all IE versions will not render the `clip-path` and this strategy produces an strange arrow. Maybe remove this fallback will be better.